### ISSUE TITLE
attribution problem when moving cases where both cases have archives

### ISSUE THREAD

**Issue by fee1-dead** [GeneralNotability/spihelper#164]
If both cases have archives, the script will delete the old archive, copy its entries to the new one while leaving a note to check the old one for attribution, and create a redirect on the old archive title.

This is bad, because the old archive should not have been deleted for attribution purposes.

See https://en.wikipedia.org/w/index.php?title=Special:Log&page=Wikipedia%3ASockpuppet+investigations%2FTahanido%2FArchive

**Comment by Sky2042** [comment]
Yes, it probably technically shouldn't do that, but archives don't really need attribution, since all the actual attribution (including the archival to some extent) already exists on the main page.
